.github/workflows/release.yml

```diff
@@ -0,0 +1,2 @@
+- name: Publish Gem Version to Internal Github Package Registry On Tag
```

Comment:
This looks like a step definition to me, which means that we're missing workflow definition e.g. `on` and the job definition